lib/_stream_readable.js

```diff
@@ -926,6 +926,32 @@ Object.defineProperty(Readable.prototype, 'readableHighWaterMark', {
   }
 });
 
+Object.defineProperty(Readable.prototype, 'readableBuffer', {
+  // making it explicit this property is not enumerable
+  // because otherwise some prototype manipulation in
+  // userland will fail
+  enumerable: false,
+  get: function() {
+    return this._readableState && this._readableState.buffer;
+  }
+});
+
+Object.defineProperty(Readable.prototype, 'readableFlowing', {
+  // making it explicit this property is not enumerable
+  // because otherwise some prototype manipulation in
+  // userland will fail
+  enumerable: false,
+  get: function() {
+    return this._readableState.flowing;
+  }
+});
+
+Readable.prototype._setFlowing = function(state) {
```

Comment:
If this  is intended to be an internal function, perhaps use a Symbol instead of a `_`-prefixed name?

Reply:
Or, if the intention is to have be accessible and used, just drop the `_`-prefix and document it.

Reply:
core (and only core) needs to use this in two places in HTTP. Where should I put that Symbol so it's accessible by HTTP?
It has an underscore because it's a private method, I'm not convinced we should make this public right now.

Another option is to conver this into a getter for `readableFlowing`.

@calvinmetcalf what do you think?

Reply:
The symbol can be dropped in to `internal/util` perhaps